cranelift-codegen/meta/src/cdsl/instructions.rs

```diff
@@ -188,6 +188,14 @@ impl Instruction {
     pub fn bind(&self, lane_type: impl Into<LaneType>) -> BoundInstruction {
         bind(self.clone(), Some(lane_type.into()), Vec::new())
     }
+
+    pub fn bind_vector(&self, lane_type: impl Into<LaneType>, num_lanes: u64) -> BoundInstruction {
```

Comment:
This allows me to do `[inst].bind(I32, 4)...`

Reply:
Nice idea.